### ISSUE TITLE
Bibtex from ADS

### ISSUE THREAD

**Issue by zilch42** [massimoaria/bibliometrix#94]
Hi there,

Is there any chance of adding support for bibtex files from ADS (astrophysical data service)? 
https://ui.adsabs.harvard.edu/

The field tags don't seem to match any of the other current database sources.

Example format:
```
@ARTICLE{2020ApSS..51045390P,
       author = {{Park}, Kyoung Ryeol and {Jeon}, Jae Eun and {Kim}, Kangmin and
         {Oh}, Nuri and {Ko}, Yong Ho and {Lee}, Jaewoong and {Lee}, Seung Hwan and
         {Ryu}, Jeong Ho and {Han}, HyukSu and {Mhin}, Sungwook},
        title = "{Synthesis of rod-type Co$_{2.4}$Mn$_{0.6}$O$_{4}$ via oxalate precipitation for water splitting catalysts}",
      journal = {Applied Surface Science},
     keywords = {Cobalt-manganese oxide, Electrocatalyst, Oxygen evolution reaction, Water splitting},
         year = 2020,
        month = apr,
       volume = {510},
          eid = {145390},
        pages = {145390},
     abstract = "{Development of a cost-effective oxygen evolution reaction (OER) catalyst
        for hydrogen production from water has attracted the attention
        of scientists due to its potential to solve current
        environmental and energy issues, such as CO$_{2}$ emissions and
        depletion of fossil fuels. In this paper, we report a facile
        synthesis to develop cobalt-manganese-oxide
        (Mn$_{x}$Co$_{3-x}$O$_{4}$, CMO) nanorods via an oxalate
        precipitation method followed by annealing at different
        temperatures. Importantly, morphology and surface area of the
        CMO nanorods, which are directly related to the OER activity,
        can be precisely controlled by changing annealing temperatures.
        The CMO nanorods engineered by oxalate precipitation and
        subsequent heat treatment show promising OER catalytic
        performance, such as a small overpotential of 365 mV for
        generating a current density of 10 mA cm$^{-2}$, a low Tafel
        slope of 50.6 mV dec$^{-1}$, and excellent long-term stability
        in alkaline media. Electrochemical properties combined with
        materials characterization provide insightful information on the
        OER mechanism of the CMO nanorods.}",
          doi = {10.1016/j.apsusc.2020.145390},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2020ApSS..51045390P},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```

**Comment by massimoaria** [comment]
At the moment, I have a very long "to do list". 
I'm hardly working to create a suite of additional packages to download bibliographic records using web API libraries from different databases. Data, that should be used with bibliometrix.
A first example is dimensionsR (still in alpha version) to gather data from Digital Science dimensions.ai (https://github.com/massimoaria/dimensionsR)
I planned to create pubmedR, scopusR, academicR (microsoft academic) and crossrefR.
Unfortunately, most of my time is engaged in academic stuffs (courses, commissions, etc.) and research activities, so I can write code only in my "free time".
In other words, any help is welcome. 
However, I'll add your request in my "to do list".

**Comment by zilch42** [comment]
Totally understand :) worth the question.

I did fork it quickly yesterday to have a look at doing it myself. It looks like a lot of regex which is never my strong suit but if I get some time I may have a deeper look into it.

Thanks
